CONTRIBUTING.md

```diff
@@ -179,6 +212,11 @@ running:
 $ go test -v cloud.google.com/go/...
 ```
 
+You can also run a specific test suite, for instance datastore be running: 
```

Comment:
"...in order to run datastore:"?

Reply:
I also am not sure if this works for directories which are not separate modules, not sure though...